test/smoke/src/areas/positron/new-project-wizard/new-project.test.ts

```diff
@@ -4,74 +4,155 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { expect } from '@playwright/test';
-import { Application, Logger } from '../../../../../automation';
+import { Application, Logger, PositronPythonFixtures } from '../../../../../automation';
 import { installAllHandlers } from '../../../utils';
 
 /*
  * New Project Wizard test cases
  */
 export function setup(logger: Logger) {
 	describe('New Project Wizard', () => {
-		// Shared before/after handling
-		installAllHandlers(logger);
-
 		describe('Python - New Project Wizard', () => {
+			// Shared before/after handling
+			installAllHandlers(logger);
+
 			before(async function () {
 
 			});
 
 			it('Python Project Defaults [C627912]', async function () {
 				const app = this.app as Application;
-				await app.workbench.positronNewProjectWizard.startNewProject();
-				await app.workbench.positronNewProjectWizard.newPythonProjectButton.click();
-				await app.workbench.positronNewProjectWizard.projectWizardNextButton.click();
-				await app.workbench.positronNewProjectWizard.projectWizardNextButton.click();
-				await app.workbench.positronNewProjectWizard.projectWizardDisabledCreateButton.isNotVisible(500); // May need to pass in a retry count > default of 200
-				await app.workbench.positronNewProjectWizard.projectWizardNextButton.click();
-				await app.workbench.positronNewProjectWizard.projectWizardCurrentWindowButton.click();
+				const pw = app.workbench.positronNewProjectWizard;
+				await pw.startNewProject();
+				await pw.projectTypeStep.pythonProjectButton.click();
+				await pw.nextButton.click();
+				await pw.nextButton.click();
+				await pw.disabledCreateButton.isNotVisible(500);
+				await pw.nextButton.click();
+				await pw.currentOrNewWindowSelectionModal.currentWindowButton.click();
 				await app.workbench.positronExplorer.explorerProjectTitle.waitForText('myPythonProject');
```

Comment:
```suggestion
				await app.workbench.positronExplorer.explorerProjectTitle.waitForText('myPythonProject');
				await app.workbench.positronConsole.waitForReady('>>>');
				await app.workbench.positronConsole.waitForConsoleContents((contents) => contents.some((line) => line.includes('restarted')));
```
I foudn that this test wasn't leaving the app in a good state. The new project was still loading, but the test finished and went on to the next test.. whcih as you pointed out, appears to not have the python extension fully loaded. This should make sure that the project from this default test if fully loaded before finishing.

Additionally in `positronConsle.waitForReady`, I would change the line that waits for the prompt to have a timeout (Github wouldln't let me add a comment to the file at the location because you hadn't changed it).. as I saw on my maching that with the above change, it would timout in `waitforReady` becuase it uses the default timout/retries.  so this change below fixed that:

```
await this.code.waitForTextContent(`${ACTIVE_CONSOLE_INSTANCE} .active-line-number`, prompt, undefined, 500)
```

Reply:
Ah, nice find 🙌 I've added in a `waitForReady` for each of the default tests and modified `waitForReady` to accept an optional retryCount with a default of 500.

The test passed in CI in the latest run! https://github.com/posit-dev/positron/actions/runs/9907461057/job/27371205186

The Jupyter test failed because IDE startup went pretty slow in CI and it timed out. I just pushed a fix and set off a new CI run. Hope it's all green this time.